hw/adsp/dsp/imx8-mu.c

```diff
@@ -51,12 +51,73 @@ static void mu_write(void *opaque, hwaddr addr,
     struct adsp_io_info *info = opaque;
     struct adsp_dev *adsp = info->adsp;
     struct adsp_reg_space *space = info->space;
+    struct qemu_io_msg_irq irq;
+    uint64_t aux;
 
     log_write(adsp->log, space, addr, val, size,
         info->region[addr >> 2]);
 
-    /* set value via SHM */
+    /* Interrupt arrived, check src */
     info->region[addr >> 2] = val;
```

Comment:
I think it is OK to leave the initial comment here ' set value via SHM' because we are actually setting the value of register via SHM. Then move the comment with 'Interrupt arrived' just above the switch.